src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```
@@ -1900,6 +1905,35 @@ paths:
         "4XX":
           $ref: "#/components/responses/ErrorResponse"
 
+  /extensions/resource-holders/page:
+    post:
+      operationId: ResourceHoldersPage
+      summary: Get Resource Holders Page
+      description: |
+        A paginated endpoint to discover which global entities hold the most of a given resource.
+        More specifically, it returns a page of global entities which hold the given resource, ordered descending by the total fungible balance / total count of non-fungibles stored in vaults in the state tree of that entity (excluding unclaimed royalty balances).
+        This endpoint operates only at the **current state version**, it is not possible to browse historical data.
+        Because of that, it is not possible to offer stable pagination as data constantly changes. Balances might change between pages being read, which might result in gaps or some entries being returned twice.
+        The default maximum page size is 1000.
```

I think this is a little confusing - people will expect the default page size to be 100. Perhaps the following is a little clearer?

```suggestion
Under default Gateway configuration, up to 100 entries are returned per response. This can be increased up to 1000 entries per page with the `limit_per_page` parameter.
``````

---

Have committed this for you to save you the hassle :)